packages/enzyme-adapter-react-16.1/src/ReactSixteenOneAdapter.js

```diff
@@ -83,9 +83,16 @@ function toTree(vnode) {
   switch (node.tag) {
     case HostRoot: // 3
       return childrenToTree(node.child);
-    case HostPortal: { // 4
-      return childrenToTree(node.child);
-    }
+    case HostPortal: // 4
+      return {
+        nodeType: 'portal',
+        type: Portal,
```

Comment:
Im not sure if it makes more sense to use `Portal` here, it may be better to just use `'Portal'`?

Reply:
typically `type` is either a string (an html element) or a function (component). I think using `the `Portal` symbol here is perfect.